internal/github/pr.go

```diff
@@ -148,3 +164,45 @@ func PRHasLabel(pr *github.PullRequest, label string) bool {
 
 	return false
 }
+
+func (ph *PRHelperImpl) GetUser(ctx context.Context, commit *object.Commit) (*github.User, error) {
+
+	email := commit.Author.Email
+	userSearchRes, _, err := ph.gc.Search.Users(ctx, email, nil)
+	if err != nil {
+		return nil, fmt.Errorf("could not get user from email %s: %v", email, err)
+	}
+	if len(userSearchRes.Users) != 1 {
+		return nil, fmt.Errorf("there is more than 1 user associated with %s: %v", email, err)
+	}
+	user := userSearchRes.Users[0]
+
+	dsOwner := ph.repoName.Owner
+	dsRepo := ph.repoName.Repo
+	filePath := "./OWNERS"
+	content, _, _, err := ph.gc.Repositories.GetContents(ctx, dsOwner, dsRepo, filePath, nil)
```

Comment:
I think we should do that only once, in the `Run` method. We should also have a local copy of the d/s repo, so getting the contents via GitHub is not necessary - just need to switch to the `main` branch.

Reply:
Fixed.